### DISCUSSION TITLE
Possible Bug in calculate_metrics_plain()

### DISCUSSION THREAD

**Discussion by Acronis1** [sebastian-hofstaetter/teaching#127] [Advanced IR: Exercise Q&A] [answered]
Hey, I submitted a PR where I clarify the issue 

https://github.com/tuwien-information-retrieval/air-23-template/pull/1

Please let me know if my changes match with the desired functionality, thanks! 

EDIT:

Seems last year there was the same issue

https://github.com/sebastian-hofstaetter/teaching/discussions/88

**Comment by pi-pa** [comment, marked as answer]
Hi, 

You're right, thx a lot!
